src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/requirements.txt

```diff
@@ -51,8 +51,8 @@ aws-cdk.cx-api==1.137
 aws-cdk.region-info==1.137
 awscli==1.22.26
 aws-sam-cli==1.36.0
```

Comment:
```bash


ERROR: Cannot install -r adf-build/requirements.txt (line 52) and boto3==1.21.31 because these package versions have conflicting dependencies.
--  
The conflict is caused by:
The user requested boto3==1.21.31
aws-sam-cli 1.15.0 depends on boto3~=1.14.23
  
To fix this you could try to:
  1. loosen the range of package versions you've specified
  2. remove package versions to allow pip attempt to solve the dependency conflict
  
ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts


```